packages/git/src/browser/git-quick-open-service.ts

```diff
@@ -133,6 +146,19 @@ export class GitQuickOpenService {
         }
     }
 
+    async pullDefault(): Promise<void> {
+        const repository = this.getRepository();
+        const remote = await this.getRemotes();
+        const defaultRemote = remote[0];
+        if (repository) {
+            try {
```

Comment:
this is a duplication of lines 179-183, is it possible to wrap it up in a separate function to reuse?